bus-mapping/src/state_db.rs

```diff
@@ -21,20 +22,10 @@ lazy_static! {
 const VALUE_ZERO: Word = Word::zero();
 
 /// Memory storage for contract code by code hash.
-#[derive(Debug, Clone)]
-pub struct CodeDB(pub HashMap<Hash, Vec<u8>>);
-
-impl Default for CodeDB {
-    fn default() -> Self {
-        Self::new()
-    }
-}
+#[derive(Debug, Clone, Default)]
```

Comment:
Since codedb is a heavy object, are we able to remove `Clone` trait for it

Reply:
It would be hard to remove `Clone` right now. Many other objects like `BlockData` and `Block` depend on it.